### PR TITLE
Initial Consideration rule 26 page

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-2016

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jsinghm13 — COMMENTED** [review]
Check if you need to give CRUD permissions to caseworker-employment-api for all fields in AuthorisationCaseField.json

File: definitions/json/CaseEventToFields.json [hunk 1]
```diff
@@ -3800,6 +3800,166 @@
...
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationRespondent",
+    "DisplayContext": "READONLY",
+    "PageID": 2,
+    "PageLabel": "Rule 26",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 1,
+    "ShowSummaryChangeOption": "Y",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:3812]: Not required, unless you want the field to display in check your answer page

File: definitions/json/CaseEventToFields.json [hunk 2]
```diff
@@ -3800,6 +3800,166 @@
...
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationRespondentLabel",
+    "DisplayContext": "READONLY",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 2,
+    "ShowSummaryChangeOption": "Y",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:3824]: Not required, unless you want the field to display in check your answer page

File: definitions/json/CaseEventToFields.json [hunk 3]
```diff
@@ -3800,6 +3800,166 @@
...
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationHearing",
+    "DisplayContext": "READONLY",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 3,
+    "ShowSummaryChangeOption": "Y",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:3835]: Not required, unless you want the field to display in check your answer page

File: definitions/json/CaseEventToFields.json [hunk 4]
```diff
@@ -3800,6 +3800,166 @@
...
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationHearingLabel",
+    "DisplayContext": "READONLY",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 4,
+    "ShowSummaryChangeOption": "Y",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:3847]: Not required, unless you want the field to display in check your answer page

File: definitions/json/CaseEventToFields.json [hunk 5]
```diff
@@ -3800,6 +3800,166 @@
...
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationJurisdictionCodes",
+    "DisplayContext": "READONLY",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 5,
+    "ShowSummaryChangeOption": "Y",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:3858]: Not required, unless you want the field to display in check your answer page

File: definitions/json/CaseEventToFields.json [hunk 6]
```diff
@@ -3800,6 +3800,166 @@
...
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "initialConsideration",
+    "CaseFieldID": "etInitialConsiderationJurisdictionCodesLabel",
+    "DisplayContext": "READONLY",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 6,
+    "ShowSummaryChangeOption": "Y",
```
Comment (jsinghm13) [inline definitions/json/CaseEventToFields.json:3870]: Not required, unless you want the field to display in check your answer page

**Review by jsinghm13 — APPROVED** [review]

**Review by jthmcts — APPROVED** [review]

**Pull request merged**